translate/asm.go

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2018 De-labtory
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package translate
+
+import "github.com/DE-labtory/koa/opcode"
+
+type output struct {
+	bytecode []byte
+	asmcode  []string
+	pc       uint64
+}
+
+func opcodeToString(p opcode.Type) string {
+	switch p {
+	case 0x01:
+		return "Add"
+	case 0x02:
+		return "Mul"
+	case 0x03:
+		return "Sub"
+	case 0x04:
+		return "Div"
+	case 0x05:
+		return "Mod"
+	case 0x10:
+		return "LT"
+	case 0x11:
+		return "GT"
+	case 0x12:
+		return "EQ"
+	case 0x13:
+		return "NOT"
+	case 0x20:
+		return "Pop"
+	case 0x21:
+		return "Push"
+	case 0x22:
+		return "Mload"
+	case 0x23:
+		return "Mstore"
+
+	default:
+		return ""
```

Comment:
I'm thinking whether return `error` or `""`(empty string) and so on. What do you think?

Reply:
I think compiler don't need to check the error about `emit`. because it's internal process of koa  whereas `vm` is independent. 
and if we should do error checking, is it more go style to return (`string`, `err`)?
```go
str, err := opcodeToString()
if err != nil {
   return err
}
```

Reply:
I'll modify this with go style! (`return (string, error)`)